pyproject.toml

```diff
@@ -68,6 +68,9 @@ omit = ['tests/*']
 [tool.pytest.ini_options]
 testpaths = ['tests', 'docs']
 
+[tool.ruff.flake8-quotes]
+inline-quotes = 'single'
+
```

Comment:
i think this was taken out deliberately before, wasn't it?

Reply:
I didn't take it out. I like the consistency of single quotes.

Reply:
it was removed here: https://github.com/WGBH-MLA/chowda/commit/9ff6f3092cd997519213f9eabf5e30a47a085ebf

rationale:
* makes no difference logically or performatively
* was causing linting errors enough to make a quick change to avoid them